src/main/java/uk/gov/ons/census/fwmt/jobservice/consumer/queue/RabbitMqConfig.java

```diff
@@ -10,32 +10,47 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Primary;
 import org.springframework.retry.RetryOperations;
+import org.springframework.retry.backoff.ExponentialBackOffPolicy;
 import org.springframework.retry.interceptor.RetryOperationsInterceptor;
+import org.springframework.retry.support.RetryTemplate;
+import uk.gov.ons.census.fwmt.common.retry.DefaultListenerSupport;
 import uk.gov.ons.census.fwmt.common.retry.GatewayMessageRecover;
+import uk.gov.ons.census.fwmt.common.retry.GatewayRetryPolicy;
 
 @Configuration
-public class QueueConfig {
-  private String username;
-  private String password;
-  private String hostname;
-  private int port;
-  private String virtualHost;
+public class RabbitMqConfig {
 
-  public QueueConfig(
+  private final String username;
+  private final String password;
+  private final String hostname;
+  private final int port;
+  private final String virtualHost;
+  private final int initialInterval;
+  private final double multiplier;
+  private final int maxInterval;
+
+  public RabbitMqConfig(
       @Value("${rabbitmq.username}") String username,
       @Value("${rabbitmq.password}") String password,
       @Value("${rabbitmq.hostname}") String hostname,
-      @Value("${rabbitmq.port}") Integer port,
-      @Value("${rabbitmq.virtualHost}") String virtualHost) {
+      @Value("${rabbitmq.port}") int port,
+      @Value("${rabbitmq.virtualHost}") String virtualHost,
+      @Value("${rabbitmq.initialInterval}") int initialInterval,
+      @Value("${rabbitmq.multiplier}") double multiplier,
+      @Value("${rabbitmq.maxInterval}") int maxInterval) {
     this.username = username;
     this.password = password;
     this.hostname = hostname;
     this.port = port;
     this.virtualHost = virtualHost;
+    this.initialInterval = initialInterval;
+    this.multiplier = multiplier;
+    this.maxInterval = maxInterval;
   }
 
-  private static CachingConnectionFactory createConnectionFactory(int port, String hostname, String virtualHost,
-      String password, String username) {
+  @Bean
```

Comment:
add the following and in RmReceiver you can change the signature to **public void receiveMessage(FieldworkFollowup fieldworkFollowup)** and not worry about converting the message. Ive used it b4 but not at ons

@Bean
    public MessageConverter jsonMessageConverter()
    {
        Jackson2JsonMessageConverter jsonMessageConverter = new Jackson2JsonMessageConverter();
        jsonMessageConverter.setClassMapper(classMapper());
        return jsonMessageConverter;
    }

   @Bean
    public DefaultClassMapper classMapper()
    {
        DefaultClassMapper classMapper = new DefaultClassMapper();
        Map<String, Class<?>> idClassMapping = new HashMap<>();
        idClassMapping.put("uk.gov.ons.census.fwmt.jobservice.dto.rm.FieldworkFollowup", FieldworkFollowup.class);
        classMapper.setIdClassMapping(idClassMapping);
        return classMapper;
    }